clustermesh-apiserver/option/config.go

```diff
@@ -52,3 +52,18 @@ var DefaultLegacyKVStoreMeshConfig = LegacyKVStoreMeshConfig{
 func (def LegacyKVStoreMeshConfig) Flags(flags *pflag.FlagSet) {
 	flags.BoolP(option.DebugArg, "D", def.Debug, "Enable debugging mode")
 }
+
+type KVStoreMeshConfig struct {
+	PerClusterReadyTimeout time.Duration
+	GlobalReadyTimeout     time.Duration
+}
```

Comment:
I didn't notice this earlier, but configurations are typically defined in the same package which registers and uses them. So in this case it would be `pkg/clustermesh/kvstoremesh`.